packages/http/src/mocker/negotiator/NegotiatorHelpers.ts

```diff
@@ -102,8 +110,9 @@ const helpers = {
     response: IHttpOperationResponse,
   ): IHttpNegotiationResult {
     const { code, dynamic, exampleKey } = partialOptions;
-    const mediaType = 'application/json';
-    const httpContent = findHttpContentByMediaType(response, mediaType);
+    const httpContent =
+      findBestHttpContentByMediaType(response, '*/*') || findBestHttpContentByMediaType(response, 'application/json');
```

Comment:
Maybe the fallback header string could be an array of strings, like `['*/*', 'application/json']` because calling this function twice will execute the lookup code twice. :)

Reply:
Done!